acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/PantheonNode.java

```diff
@@ -67,14 +67,13 @@
 import org.web3j.protocol.websocket.WebSocketService;
 import org.web3j.utils.Async;
 
-public class PantheonNode implements Node, NodeConfiguration, RunnableNode, AutoCloseable {
```

Comment:
Is any class still using `Node` interface, once removed from `PantheonNode`?

Reply:
Yes, `PantheonNode`.  `RunnableNode` extends `Node` so no point in declaring it twice (it really messes up class hierarchy views).

Reply:
👍